jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -252,87 +312,58 @@ static void afterClinitInitialization() {
 		StringCoding.encode(new char[1], 0, 1);
 		/*[ENDIF]*/
 
-		/*[IF Sidecar18-SE-OpenJ9 | JAVA_SPEC_VERSION >= 11]*/
+		/*[IF Sidecar18-SE-OpenJ9]*/
 		Properties props = internalGetProperties();
 		/*[IF JAVA_SPEC_VERSION >= 11]*/
 		consoleDefaultCharset = Charset.defaultCharset();
```

Comment:
It seems assumed that all usages of `consoleDefaultCharset` are after the initialization here, a comment around the variable declaration might avoid confusion.

Reply:
I've removed getConsoleDefaultEncoding() and getConsoleDefaultCharset() which are no longer used and added some comments.